### DISCUSSION TITLE
After the image is reduced, the image lineweight is not displayed. For the desired effect, infinitely reduce the line width of at least 1 pixel

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#8194] [Q&A]
After the image is reduced, the image lineweight is not displayed. For the desired effect, infinitely reduce the line width of at least 1 pixel
Please look at 3 picture examples
Picture1:
![1](https://user-images.githubusercontent.com/89827437/154783667-a1821cbc-72e1-4570-b9af-aab179fef662.png)

Picture2:
![2](https://user-images.githubusercontent.com/89827437/154783672-de3bd631-735c-4574-92b6-bc428c0f9247.png)

Picture3:
![3](https://user-images.githubusercontent.com/89827437/154783677-5370d7c1-0e22-4ed4-8d98-82225b812649.png)

**Comment by bigtimebuddy** [comment]
Two options:

1. Divide your line style width by the scale: e.g. `10 / 0.0001` this will make it appear as if there are no changes. If you want to limit the size, you can use `Math.min`, say if you don't want the width to go above 1000.
2. Consider drawing with native lines (`native: true` option in lineStyle). This will ignore the world scale and always render at 1 pixel. The trade-off is no anti-aliasing or smoothing.

**Reply by pcbren1987** [reply]
Thank you for your answer. I want to know the drawing principle of pixijs. If you reduce the scale by 0.001, the line is visible. If you reduce the scale by 0.0001, the line is invisible.

Is there any way to reduce 0.0001 or 0.00001 visible? Is there any way

**Reply by bigtimebuddy** [reply]
You are somewhat at the whim of the GPU and what it chooses to render. There's not a great way to represent those marginal changes and I'm sure it's different for different GPUs. You may want to check out graphics-smooth a PixiJS plugin that does a novel approach to rendering subpixel strokes. You may get more consistent results.
